src/webui/www/webui.qrc

```diff
@@ -17,6 +17,7 @@
         <file>private/css/Window.css</file>
         <file>private/download.html</file>
         <file>private/downloadlimit.html</file>
+        <file>private/edit_feed_url.html</file>
```

Comment:
```suggestion
        <file>private/editfeedurl.html</file>
```
I'm not sure we have naming rule for files.. @glassez any ideas?

Reply:
Main sources don't use underscores in filenames so I would follow this implicit rule.